<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtrudePropertrDialog</class>
 <widget class="QDialog" name="ExtrudePropertrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="widthEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="upEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="downEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="propertyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>属性</string>
   </property>
  </widget>
  <widget class="QLabel" name="upCommitLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>负凸正凹</string>
   </property>
  </widget>
  <widget class="QLabel" name="downCommitLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>正凸负凹</string>
   </property>
  </widget>
  <widget class="QLabel" name="downLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QLabel" name="upLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
